core-domain/src/main/java/org/uniprot/core/cv/xdb/impl/CrossRefEntryImpl.java

```diff
@@ -41,7 +42,7 @@ public CrossRefEntryImpl(
         this.pubMedId = pubMedId;
         this.doiId = doiId;
         this.linkType = linkType;
-        this.server = server;
+        this.servers = servers;
```

Comment:
Can you please use this.servers = Utils.unmodifiableList(servers) in the constructor